src/pretix/base/models/event.py

```diff
@@ -634,6 +634,11 @@ def copy_data_from(self, other):
             Quota,
         )
 
+        # Note: avoid self.set_active_plugins() causes trouble for BadgeApp plugin
+        #  Plugins can create data in installed() hook based on existing data of the event.
+        #  Calling set_active_plugins() results in defaults being created while actually data
+        #  should come from the copied event. Instead events should use they might be copied
+        #  using the event_copy_data-signal for new events.
```

Comment:
Can you… read over this comment again? It looks like three incomplete sentences.